### PR TITLE
regenerated languages

### PR DESCRIPTION
**Pull request by cdietrich**
regenerated languages
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java [hunk 1]
```diff
@@ -770,8 +776,8 @@ public ParserRule getXListLiteralRule() {
 	//XClosure XExpression:
 	//	=> ({XClosure}
 	//	'[')
-	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)?
-	//	explicitSyntax?='|')?
+	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)
+	//	? explicitSyntax?='|')?
```
Comment (szarnekow) [inline org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java:780]: Hmm this is really a bad mis-formatting. There should never be a space before the cardinality according to https://github.com/eclipse/xtext-core/blob/master/org.eclipse.xtext/src/org/eclipse/xtext/xtext/XtextFormatterJava.java#L204

Comment (cdietrich) [reply org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java:780]: maybe we miss some traverals

Comment (cdietrich) [reply org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java:780]: @ArneDeutsch this one still looks bad

**Review by szarnekow — APPROVED** [review]

File: org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java (hunk 1 above)
Comment (ArneDeutsch) [reply org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java:780]: Will be fixed with next commit. Allow autowrap after question mark as well.

**Pull request merged**